ember-cli-build.js

```diff
@@ -21,27 +23,52 @@ module.exports = function(defaults) {
 };
 
 
+/** Generate RTL CSS files along side regular CSS. */
+class BroccoliSassWithRtl extends BroccoliSass {
+  build() {
+    // This will output the regular CSS.
+    super.build();
+
+    // We then read that file and output the RTL CSS.
+    const cssOutputPath = path.join(this.outputPath, this.outputFile);
+    const cssRtlOutputPath = path.join(this.outputPath,
+        path.dirname(this.outputFile),
+        path.basename(this.outputFile, '.css') + '-rtl.css');
+    fs.writeFileSync(cssRtlOutputPath, cssjanus.transform(fs.readFileSync(cssOutputPath).toString()));
+  }
+}
+
+
+
 /** Gets the tree for all of the components' CSS. */
-function getComponentsCssTree() {
-  // Assume that the list of components is all directories in `src/components/`
-  var componentsSrc = 'src/components/';
-  var components = fs.readdirSync(componentsSrc)
-      .filter(file => fs.statSync(path.join(componentsSrc, file)).isDirectory());
-
-  var componentCssTrees = components.reduce((trees, component) => {
-    // We want each individual scss file to be compiled into a corresponding css file
-    // so that they can be individually included in styleUrls.
-    var scssFiles = fs.readdirSync(path.join(componentsSrc, component))
-        .filter(file => path.extname(file) === '.scss')
-        .map(file => path.basename(file, '.scss'));
-
-    return scssFiles.map(fileName => {
-      return BroccoliSass(
-        [`src/components/${component}`, 'src/core/style'], // Directories w/ scss sources
-        `./${fileName}.scss`,                              // Root scss input file
-        `components/${component}/${fileName}.css`);        // Css output file
-    }).concat(trees);
-  }, []);
-
-  return broccoliAutoprefixer(mergeTrees(componentCssTrees), autoprefixerOptions);
+function getCssTree(source, destination) {
+  function walk(dir) {
```

Comment:
Move the `walk` function out of the function body of `getCssTree` and add a description.